flask/app/models.py

```diff
@@ -76,16 +76,63 @@ class Participant(db.Model):
 
 
 class TissueSampleType(Enum):
-    Blood = 'Blood'  # BLO
```

Comment:
The _name_ of the enumeration is used by SQLAlchemy in the database and the _value_ only has an effect on the Python side. This means that when serialized to SQL, the names will be Pascal_Snake_Case but when serialized to JSON, the names with be "human readable - with spaces". I think this is confusing and we should just use the same names on both sides.

Reply:
Hmm, that's a good point. I guess the only problem that arises is when we try to map these to the GTEx samples for the rnaseq pipeline (i.e. should we do a mapping or change the pipeline reference). Since several of them have special characters (parentheses and dashes) they may not be automatically map-able.

This makes me wonder whether it's actually better to just eliminate references to the GTEx database in the Tissue table and make it a RNASeq dataset specific attribute. Or something selected at analysis runtime. Thoughts?